toolkit/resources/manifests/package/local.repo

```diff
@@ -18,7 +20,7 @@ skip_if_unavailable=1
 sslverify=0
 priority=2
 
-# Then packages we have already cached from upstream...
+# To satisfy all remaining dependencies, we use the upstream packages.
```

Comment:
As I recall this isn't actually upstream yet, just the stuff already in the cache. Name is maybe not great .

Reply:
Ah, right, I keep getting lost in this part of the cloner/fetcher logic. Let me try to write something better.